### PR TITLE
feat: Fix Metastore CVEs reported by security team

### PR DESCRIPTION
**Pull request by sumandas0**
## Security issues reported by Security team on few dependencies, mostly are on hadoop versions and Ranger dependencies JARs

> Security issues reported by Security team on few dependencies, mostly are on hadoop versions and Ranger dependencies JARs, removed all the Ranger dependencies at once and also removed Hadoop auth dependencies. Tried to remove All Hadoop dependencies all together but multiple dependencies are there.

Actions
- Removed Ranger plugin dependency
- Removed dependency solr-solrj - 7.7.1
- Removed Hadoop auth  from Atlas.

## Type of change
- [x] Security fix (fixes an  security issue)
- [ ] New feature (adds functionality)

## Related issues

> Fixes [Fix vulnerabilities in dependencies of atlas-metastore](https://atlanhq.atlassian.net/browse/PLT-474) 

### Development

- [ ] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java [hunk 1]
```diff
@@ -130,7 +130,7 @@ public GraphBackedSearchIndexer(AtlasTypeRegistry typeRegistry) throws AtlasExce
         this.typeRegistry = typeRegistry;
 
         //make sure solr index follows graph backed index listener
-        addIndexListener(new SolrIndexHelper(typeRegistry));
+        //addIndexListener(new SolrIndexHelper(typeRegistry));
```
Comment (checkaayush) [inline repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java:133]: Why is this commented and not removed?

**Review by checkaayush — APPROVED** [review]
Left a minor comment. Rest LGTM.

File: repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java (hunk 1 above)
Comment (sumandas0) [reply repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java:133]: We can remove it directly, but just letting it there to have traceability

Comment (checkaayush) [reply repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java:133]: Let got take care of traceability and version control. Git history stays preserved, so tracking it via that would still be possible if we remove it.

Comment (sumandas0) [reply repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java:133]: I think we can remove it.

**Review by checkaayush — APPROVED** [review]

File: repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java (hunk 1 above)
Comment (sumandas0) [reply repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java]: Removed it

**Pull request merged**